django/forms/utils.py

```diff
@@ -42,19 +44,40 @@ def flatatt(attrs):
     )
 
 
+DEFAULT_TEMPLATE_DEPRECATION_MSG = """
+The "default.html" templates for forms and formsets will be removed. These were
+proxies to the equivalent "table.html" templates, but the new "div.html"
+templates will be the default from Django 5.0.
+
+Transitional renderers are provided to allow you to opt-in to the new output
+style now. For example, set:
+
+    FORM_RENDERER = "django.forms.renderer.DjangoDivFormRenderer"
+
+… in your settings file, or provide your own appropriate renderer class.
+
+See https://docs.djangoproject.com/en/4.1/releases/4.1/ for more details.
```

Comment:
This should use `get_docs_version()` instead of 4.1:
```python
"""
...
See https://docs.djangoproject.com/en/%s/releases/4.1/ for more details.
""" % get_docs_version()
```

Reply:
> Maybe it could be just:
> 
> ```
> The "default.html" templates for forms and formsets will be removed.
> See https://docs.djangoproject.com/en/4.1/releases/4.1/ for more details.
> ```

... or at least remove an example:
```
The "default.html" templates for forms and formsets will be removed. These were
proxies to the equivalent "table.html" templates, but the new "div.html"
templates will be the default from Django 5.0.
Transitional renderers are provided to allow you to opt-in to the new output
style now. See https://docs.djangoproject.com/en/%s/releases/4.1/ for more details.
```

Reply:
I went for the last option 🙂